src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsLambda/Instrumentation.xml

```diff
@@ -17,5 +17,11 @@ SPDX-License-Identifier: Apache-2.0
         <exactMethodMatcher methodName=".ctor" />
       </match>
     </tracerFactory>
+    <!-- Instrument the function handler for AspNetCore lambdas -->
+    <tracerFactory name="NewRelic.Providers.Wrapper.AwsLambda.HandlerMethod">
+      <match assemblyName="Amazon.Lambda.AspNetCoreServer" className="Amazon.Lambda.AspNetCoreServer.AbstractAspNetCoreFunction`2">
```

Comment:
If you have time, maybe we could try a couple older versions to see if we need to use version filtering? Or maybe you've already done that.

Reply:
Tested v7.0.1 which is the oldest that targets .NET and dates back to the end of 2021, no issues. Current release is v9.0.0.